### PR TITLE
Define project for community.yang

### PR DESCRIPTION
**Pull request by GomathiselviS**
Signed-off-by: GomathiselviS <gomathiselvi@gmail.com>

### CODE REVIEW

**Comment by GomathiselviS** [comment]
recheck

File: zuul.d/projects.yaml [hunk 1]
```diff
@@ -227,6 +227,20 @@
...
+    templates:
+      - system-required
+      - ansible-collections-iosxr-community-yang
+      - ansible-collections-junos-community-yang
+      - ansible-python-jobs
+      - integrated-queue
+      - publish-to-galaxy
+      - publish-to-automation-hub
+
+
```
Comment (pabelanger) [inline zuul.d/projects.yaml:243]: extra spaces can be removed

File: zuul.d/projects.yaml [hunk 2]
```diff
@@ -227,6 +227,20 @@
...
+- project:
+    name: github.com/ansible-collections/community.yang
+    default-branch: main
+    merge-mode: squash-merge
+    templates:
+      - system-required
+      - ansible-collections-iosxr-community-yang
+      - ansible-collections-junos-community-yang
+      - ansible-python-jobs
+      - integrated-queue
```
Comment (pabelanger) [inline zuul.d/projects.yaml:239]: can be removed, for now

File: zuul.d/projects.yaml [hunk 3]
```diff
@@ -227,6 +227,20 @@
     templates:
       - publish-to-galaxy-3pci
 
+- project:
+    name: github.com/ansible-collections/community.yang
+    default-branch: main
+    merge-mode: squash-merge
+    templates:
+      - system-required
+      - ansible-collections-iosxr-community-yang
```
Comment (pabelanger) [inline zuul.d/projects.yaml:236]: lets remove and only add units here first.

File: zuul.d/projects.yaml [hunk 4]
```diff
@@ -227,6 +227,17 @@
     templates:
       - publish-to-galaxy-3pci
 
+- project:
+    name: github.com/ansible-collections/community.yang
+    default-branch: main
+    merge-mode: squash-merge
+    templates:
+      - system-required
+      - ansible-collections-community-yang-units
```
Comment (pabelanger) [inline zuul.d/projects.yaml:236]: lets remove this for now, so we can get some jobs working

File: zuul.d/projects.yaml [hunk 5]
```diff
@@ -227,6 +227,16 @@
...
 
+- project:
+    name: github.com/ansible-collections/community.yang
+    default-branch: main
+    merge-mode: squash-merge
+    templates:
+      - system-required
+      - ansible-python-jobs
+      - publish-to-galaxy
+      - publish-to-automation-hub
```
Comment (pabelanger) [inline zuul.d/projects.yaml:238]: actually, this cannot go onto automation hub, it is community so galaxy

**Pull request merged**